k-distribution/tests/regression-new/pedanticAttributes/pedanticAttributes.k.out

```diff
@@ -0,0 +1,6 @@
+[Error] Compiler: Unrecognized attributes: [badAtt]
```

Comment:
You could specify in the error message that: user defined attributes can be added with `group(my, atts)`